test cases/rust/12 bindgen/meson.build

```diff
@@ -1,7 +1,7 @@
 # SPDX-license-identifer: Apache-2.0
 # Copyright © 2021-2022 Intel Corporation
```

Comment:
Do you care about updating the copyright here? I think the project should just remove date ranges and keep the first one.